arviz/tests/test_data.py

```diff
@@ -30,6 +34,60 @@ def chains():
     return 2
 
 
+@pytest.fixture(autouse=True)
+def no_remote_data(tmpdir):
+    """Delete all remote data and replace it with a local dataset."""
+    keys = list(REMOTE_DATASETS)
+    for key in keys:
+        del REMOTE_DATASETS[key]
+
+    centered = LOCAL_DATASETS["centered_eight"]
+    filename = os.path.join(str(tmpdir), os.path.basename(centered.filename))
+    url = "file://" + centered.filename
```

Comment:
No change required. `+` operator string formatting is an antipattern imo. Might be better to use `os.path.join` to avoid OS specific problems.

Reply:
True! I ended up using `urlunsplit`, since the _scheme_ is `file`. I think this should make it OS portable.